python/cudf_kafka/cudf_kafka/_lib/kafka.pxd

```diff
@@ -34,7 +34,7 @@ cdef extern from "kafka_consumer.hpp" \
         int64_t get_committed_offset(string topic,
                                      int32_t partition) except +
 
-        map[string, vector[int32_t]] list_topics() except +
+        map[string, vector[int32_t]] list_topics(string topic) except +*
```

Comment:
```suggestion
        map[string, vector[int32_t]] list_topics(string topic) except +
```

Should just be `except +`, since this function can never raise Python exceptions